### DISCUSSION TITLE
Does it exist bug with @Step annotation?

### DISCUSSION THREAD

**Discussion by DejanPerovic** [allure-framework/allure2#1445] [Bug Reports] [answered]
I use 2.15.0 version in Maven project in combination with Selenium and TestNG. All other annotations work well except @Step. I do not get any info on the report.
Does it exist some bug?

**Comment by baev** [comment, marked as answer]
it works for me in my tests. Probably you have a configuration issue

**Reply by DejanPerovic** [reply]
Could you please provide me with your configuration?
I don't get why the server works well and generates reports with all other annotations which I put such as @Epic, @Future, @Story, @Severity,...

In pom.xml I have this:

```
<properties>
  <allure-version>2.15.0</allure-version>
</properties>

....
  <!-- https://mvnrepository.com/artifact/io.qameta.allure/allure-testng -->
        <dependency>
            <groupId>io.qameta.allure</groupId>
            <artifactId>allure-testng</artifactId>
            <version>${allure-version}</version>
        </dependency>
```

**Reply by baev** [reply]
`@Step` and `@Attachment` annotations requires AspectJ configuration while other annotation isn't. 

You can find the working example here https://github.com/allure-examples/allure-examples

BTW the AspectJ configuration https://github.com/allure-examples/allure-examples/blob/master/allure-testng/pom.xml#L50

**Reply by DejanPerovic** [reply]
Thank you very much baev! It works now! Somehow I missed that.
